src/SDKs/DataLake.Analytics/Management.DataLake.Analytics/Properties/AssemblyInfo.cs

```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure Data Lake Analytics management operations including account, catalog and job management.")]
 
 [assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.2.1.0")]
+[assembly: AssemblyFileVersion("2.2.2.0")]
```

Comment:
@begoldsm bump up to 3.0.0, because your last stable was 2.0.0 and this seems to have breaking change compared to 2.0.0